service/history/timerQueueActiveProcessor.go

```diff
@@ -74,6 +74,7 @@ func newTimerQueueActiveProcessor(shard ShardContext, historyService *historyEng
 		timeNow,
 		updateShardAckLevel,
 		logger,
+		currentClusterName,
```

Comment:
do you actually use this?

Reply:
Yes I need them otherwise code cannot run